src/tests/ftest/util/dmg_utils.py

```diff
@@ -290,13 +291,14 @@ def pool_create(path, host_port, scm_size, insecure=True, group=None,
     return result
 
 
-def pool_destroy(path, host_port, pool_uuid, insecure=True, force=True):
+def pool_destroy(path, pool_uuid, host_port=None, insecure=True, force=True):
```

Comment:
Do we not have any other files that call this function?  I would expect to also see changes where this method is called do to the argument order change.  Just confirming this has been considered.

Reply:
No. `create_dmg` is the only place that uses this function and I have updated it.

Reply:
Ok, good. Thanks.